packages/appstore/src/components/wallet-content/wallet-option-multipliers-listing.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Text, StaticUrl } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import ListingContainer from 'Components/containers/listing-container';
+import TradingAppCard from 'Components/containers/trading-app-card';
+import PlatformLoader from 'Components/pre-loader/platform-loader';
+import { getHasDivider } from 'Constants/utils';
+import { useStore, observer } from '@deriv/stores';
+import { useActiveWallet } from '@deriv/hooks';
+import './wallet-content.scss';
+
+const WalletOptionsAndMultipliersListing = observer(() => {
+    const { traders_hub, client, ui } = useStore();
+    const { is_mobile, setShouldShowCooldownModal, openRealAccountSignup } = ui;
+    const {
+        is_landing_company_loaded,
+        has_maltainvest_account,
+        real_account_creation_unlock_date,
+        is_logging_in,
+        is_switching,
+    } = client;
+    const { available_platforms, is_eu_user, is_real, no_MF_account, no_CR_account, is_demo } = traders_hub;
+
+    const wallet_account = useActiveWallet();
+
+    if (!wallet_account || is_switching || is_logging_in || !is_landing_company_loaded)
+        return (
+            <div className='wallet-content__loader'>
+                <PlatformLoader />
+            </div>
+        );
+
+    const OptionsTitle = () => {
+        if (wallet_account.landing_company_name === 'svg' && !is_mobile) {
+            return (
+                <Text size='sm' line_height='m' weight='bold' color='prominent'>
+                    {localize('Options & multipliers')}
+                </Text>
+            );
+        } else if (wallet_account.landing_company_name !== 'svg' && !is_mobile) {
+            return (
+                <Text size='sm' line_height='m' weight='bold' color='prominent'>
+                    {localize('Multipliers')}
+                </Text>
+            );
+        }
+        return null;
+    };
+
+    const listing_container_description =
+        wallet_account.landing_company_name === 'svg' ? (
```

Comment:
could we add a constant variable for this check `wallet_account.landing_company_name === 'svg'` its used in other places as well in this file